docs/migration/v8-to-v9.md

```diff
@@ -80,19 +80,21 @@ In v9, an `undefined` value will be treated the same as if the value is not defi
 
 ### `@sentry/node`
 
-- When `skipOpenTelemetrySetup: true` is configured, `httpIntegration({ spans: false })` will be configured by default. This means that you no longer have to specify this yourself in this scenario. With this change, no spans are emitted once `skipOpenTelemetrySetup: true` is configured, without any further configuration being needed.
+- When `skipOpenTelemetrySetup: true` is configured, `httpIntegration({ spans: false })` will be configured by default.
+
+This means that you no longer have to specify this yourself in this scenario. With this change, no spans are emitted once `skipOpenTelemetrySetup: true` is configured, without any further configuration being needed.
 
 ### `@sentry/browser`
 
-- The `captureUserFeedback` method has been removed. Use `captureFeedback` instead and update the `comments` field to `message`.
+- The `captureUserFeedback` method has been removed. Use the `captureFeedback` method instead and update the `comments` field to `message`.
 
 ### Uncategorized (TODO)
 
 TODO
 
 ## 3. Package Removals
 
-As part of an architectural cleanup we deprecated the following packages:
+As part of an architectural cleanup, we deprecated the following packages:
```

Comment:
l: Once v9 ships, we won't publish utils and types anymore, right? 

```suggestion
As part of an architectural cleanup, we deprecated and stopped publishing the following packages:
```

Reply:
We'll still keep publishing types but not utils